packages/mosaic/core/src/MosaicClient.ts

```diff
@@ -23,7 +29,7 @@ export class MosaicClient {
   _filterBy: Selection | undefined;
   _requestUpdate: (event?: unknown) => void;
   _coordinator: Coordinator | null;
-  _pending: Promise<any>;
+  _pending: Promise<unknown>;
```

Comment:
unknown means that a user won't be able to really know anything about the type. Maybe we want to make this a generic?

Reply:
Actually, this should always be `Promise<void>`. Fixed!